scripts/ci/all.sh

```diff
@@ -4,6 +4,6 @@ srcdir=`dirname $0`
 test -z "$srcdir" && srcdir=.
 . ${srcdir}/common.sh
 
-DEFAULT_COMPOSE_ARGS="master --no-apm-server-dashboards --no-apm-server-self-instrument --with-agent-rumjs --with-agent-go-net-http --with-agent-nodejs-express --with-agent-python-django --with-agent-python-flask --with-agent-ruby-rails --with-agent-java-spring --force-build --build-parallel"
+DEFAULT_COMPOSE_ARGS="${ELASTIC_STACK_VERSION:-'master'} ${BUILD_OPTS} --no-apm-server-self-instrument --no-apm-server-dashboards --with-agent-rumjs --with-agent-go-net-http --with-agent-nodejs-express --with-agent-python-django --with-agent-python-flask --with-agent-ruby-rails --with-agent-java-spring --force-build"
```

Comment:
when is `ELASTIC_STACK_VERSION` set?

Reply:
`ELASTIC_STACK_VERSION` and `BUILD_OPTS` can be set before launching the script, if not, `ELASTIC_STACK_VERSION` will be ignored and the script will use `master` as Elastic Stack version, BUILD_OPTS will be blank and will have no effect.

But, I am thinking that I can move ELASTIC_STACK_VERSION to common.sh, probably is more clear, also I will add some kind of header explaining what those variables are for.

Reply:
I have added some help to the readme.